src/tasks/twitchRefreshSubscriptions.ts

```diff
@@ -14,7 +15,7 @@ export default class extends Task {
 		if (this.client.options.dev) return;
 
 		// Retrieve all the Twitch subscriptions
-		const allSubscriptions = await this.client.queries.fetchAllTwitchStreams();
+		const allSubscriptions = await (await DbSet.connect()).twitchStreamSubscriptions.find();
```

Comment:
Please..no more `await (await)`. Extract the variable, and pass it down to `updateEntries` as well, skip one connect call. OR HELL, pass the `twitchStreamSubscriptions` repository over